packages/web-app-files/src/mixins/actions/emptyTrashBin.js

```diff
@@ -41,7 +41,7 @@ export default {
         .catch((error) => {
           console.error(error)
           this.showMessage({
-            title: this.$gettext('Could not delete files'),
+            title: this.$gettext('Failed to delete all files permanently'),
```

Comment:
Should we add context about "all files in the trashbin" here? Message sounds kinda scary otherwise :)

Reply:
Good idea!